FHEM/14_SD_UT.pm

```diff
@@ -1978,6 +2013,11 @@ sub SD_UT_Attr {
           $deviceCode = substr($bitData,0,12);
           $deviceCode = sprintf("%03X", oct( "0b$deviceCode" ) );
           $devicename = $devicemodel.'_'.$deviceCode;
+        ############ BeSmart_S4 ############
+        } elsif ($attrValue eq 'BeSmart_S4') {
```

Comment:
Anscheinen kam kein Test jemals an diese Stelle. Ich kann das jetzt nicht exakt einordnen, aber grundsätzlich haben wir die Testdaten vermutlich im Repo SIGNALDUINO_TOOL dafür oder?

Reply:
Testdaten für das Protokoll habe ich sowohl in prerelease als auch in master beim SIGNALDUINO_TOOL eingepflegt.

Reply:
Gleiches wie bei set, Attribute werden beim Test nicht verändert. Also auch hier kein neues Problem.